subprojects/common/report-viewer/src/main/kotlin/com/avito/report/internal/ReportsAddApiImpl.kt

```diff
@@ -213,12 +209,12 @@ internal class ReportsAddApiImpl(private val client: JsonRpcClient) : ReportsAdd
             "environment" to test.device.name,
             "report" to report,
             // тут происходит магия "с помощью оператора добавляется в массив новое значение билда"
-            // todo узнать у Жени как можно это лучше сделать
             "report_data" to reportData,
             // todo onlyIf present
             "console" to mapOf(
-                "stdout" to stdout,
-                "stderr" to stderr
+                "stdout" to logcat,
+                // non empty value causes warning sign to appear in UI
+                "stderr" to ""
```

Comment:
How does it look?
Should we express it explicitly for a client instead of empty value?

Reply:
![image](https://user-images.githubusercontent.com/1105133/120382679-9a526300-c32c-11eb-8d14-705bb59a0653.png)

Like this. We can but it will be an invalid link probably

Reply:
I wanted to add "All logcat output goes to stdout" message to all stderr's, but it seems like a huge waste of traffic. It will be added to tens of thousands tests a day... 
Should be probably handled on report frontend side